### ISSUE TITLE
Config options

### ISSUE THREAD

**Issue by jjohnsnaill** [Lykrast/MeetYourFight#2]
There needs to be a way to configure various things in the mod. Some examples:

Item stats. They won't be suitable for every mod pack, and should have config options.

Boss stats and behaviour. For example, I feel Swampjaw is quite weak, even on Hard, and would like to be able to configure its health, speed, and how often it drops mines. Maybe also a frequency multiplier that's gradually applied as it loses health, and separate options for each difficulty setting.

**Comment by Lykrast** [comment]
I don't think I can realistically give in-depth config options, so no guarantee for that. Though if there are specific items that you think are too weak/too strong relative to how easy/hard they are to get, feel free to let me know.

> For example, I feel Swampjaw is quite weak

Also, Swampjaw is meant to be an easy boss fight, since you can fight it as soon as you find any hostile mobs.

**Comment by jjohnsnaill** [comment]
> 
> Also, Swampjaw is meant to be an easy boss fight, since you can fight it as soon as you find any hostile mobs.

True, though packs may want to modify the summoning item's recipe to move it later into progression.

**Comment by Lykrast** [comment]
I mean if they want to move the easy boss much later in progression that's the pack's problem.

**Comment by Lykrast** [comment]
added next update
